rules/library.bzl

```diff
@@ -419,17 +435,48 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         private_deps.append(linkopts_name)
 
     vendored_deps = []
+    import_vfsoverlays = []
+
+    # Note: for these, we'll need to output the VFS rule for each of the imports
+    # The depend on that rule in the top level one
     for vendored_static_framework in kwargs.pop("vendored_static_frameworks", []):
         import_name = "%s-%s-import" % (name, paths.basename(vendored_static_framework))
         apple_static_framework_import(
             name = import_name,
             framework_imports = native.glob(
                 ["%s/**/*" % vendored_static_framework],
                 exclude = ["**/.DS_Store"],
+                # Note: there are some frameworks imported with private headers
+                #"%s/PrivateHeaders" % vendored_static_framework],
```

Comment:
Nit: Could we move the comment up to line 446?

Reply:
Why would this be more suited there?

Reply:
I thought the comment explained why you glob `["%s/**/*" % vendored_static_framework]` and generally prefer writing comments before the code.

Reply:
Yep I think this suggestion makes a lot of sense - I'll move the comment up a bit!